core/src/main/java/org/frankframework/validation/RootValidation.java

```diff
@@ -40,7 +39,7 @@ public RootValidation(String... rootElement) {
 	public void check(IConfigurationAware source, Set<IXSD> xsds) {
 		boolean found = false;
 		String validElements = rootValidation.get(rootValidation.size() - 1);
-		List<String> validElementsAsList = Arrays.asList(validElements.split(","));
+		String[] validElementsAsList = validElements.split(",");
 		for (String validElement : validElementsAsList) {
```

Comment:
Tja.. beetje zeurpuntje zo. maar ik doe het stiekem toch
```suggestion
		List<String> validElementsAsList = StringUtil.split(validElements);
		for (String validElement : validElementsAsList) {
```